cmd/tracee-ebpf/flags/capabilities.go

```diff
@@ -5,116 +5,71 @@ import (
 	"strings"
 
 	"github.com/aquasecurity/tracee/pkg/capabilities"
-	"github.com/aquasecurity/tracee/pkg/filters"
-	"kernel.org/pub/linux/libs/security/libcap/cap"
-)
-
-const (
-	CapsMainFlag        = "caps"
-	AllowFailedDropFlag = "allow-failed-drop"
-	CancelDropFlag      = "cancel-drop"
-	AddReqCapsFlag      = "add"
+	tracee "github.com/aquasecurity/tracee/pkg/ebpf"
 )
 
 func capabilitiesHelp() string {
-	mainHelp := fmt.Sprintf(`Manage the capabilities and capabilities-related operations of tracee-ebpf
-Normally, tracee will drop all capabilities not required to its operations.
 
-Possible options:
-  %-51s allow tracee-ebpf to run with high capabilities, in case that capabilities dropping fails
-  %-51s cancel capabilities drop, so tracee will run with all given capabilities (unless missing required capabilities)
-  %-51s add required capabilities, so tracee won't drop them`,
-		AllowFailedDropFlag,
-		CancelDropFlag,
-		fmt.Sprintf("%s=<list_of_caps>", AddReqCapsFlag))
+	availCaps := strings.Join(capabilities.ListAvailCaps(), "\n  ")
 
-	exampleUses := map[string]string{
-		"--%[1]s %[2]s":                          "Try capabilities drop, but continue run if failed",
-		"--%[1]s %[3]s":                          "Don't do capabilities drop",
-		"--%[1]s %[4]s=cap_syslog":               "Drop all required capabilities except for required and CAP_SYSLOG",
-		"--%[1]s %[4]s!=cap_sys_ptrace":          "Drop all capabilities except CAP_SYS_PTRACE, unless required",
-		"--%[1]s %[2]s --%[1]s %[4]s=cap_syslog": "Try drop all capabilities except for required and CAP_SYSLOG, but continue run if failed",
-	}
+	return `
+Opt out from dropping capabilities by default OR set specific ones.
 
-	var exampleUsesString string
-	for cmd, exp := range exampleUses {
-		cmd = fmt.Sprintf(cmd, CapsMainFlag, AllowFailedDropFlag, CancelDropFlag, AddReqCapsFlag)
-		exampleUsesString += fmt.Sprintf("  %-65s | %s\n", cmd, exp)
-	}
-
-	return fmt.Sprintf("%s\n\nExample uses:\n%s", mainHelp, exampleUsesString)
-}
+Possible options:
+  --capabilities bypass={0,1}               | keep all capabilities during execution time.
```

Comment:
can we use `bypass=[true|false]`? At least for me I will bug every time I have to think which enables, which doesn't.